http/design/body_types.go

```diff
@@ -187,6 +189,36 @@ func buildResponseBody(name string, attr *design.AttributeExpr, resp *HTTPRespon
 	return &design.AttributeExpr{Type: nmt, Validation: userType.Validation}
 }
 
+func setForcePointer(att *design.AttributeExpr, seen ...map[string]struct{}) {
```

Comment:
Is it really necessary to set the flag on all attributes? seems like it's only needed if the attribute has a primitive type? (so an additional case for primitive types that sets the flag might be enough?)

Reply:
Fixed.